src/js/stream.js

```diff
@@ -63,30 +63,30 @@ class Comments {
 	}
 
 	/**
-	 * Get the data attributes from the CommentsElement. If the component is being set up
+	 * Get the data attributes from the StreamElement. If the component is being set up
 	 * declaratively, this method is used to extract the data attributes from the DOM.
 	 *
-	 * @param {HTMLElement} oCommentsEl - The component element in the DOM
+	 * @param {HTMLElement} streamEl - The component element in the DOM
 	 * @returns {Object} - Data attributes as an object
 	 */
-	static getDataAttributes (oCommentsEl) {
-		if (!(oCommentsEl instanceof HTMLElement)) {
+	static getDataAttributes (streamEl) {
```

Comment:
`getDataAttributes` is repeated in `Count` and `Stream`, that's ok but we could reduce the bundle size a little by getting data attributes in one place. Maybe the main `Comments` class would be a good place? It could get the data attributes and pass to either `Count` or `Stream`.

Reply:
I like the idea of moving `getDataAttributes` into `main.js`. Avoid repeating a function that is identical in both files. I'll work on this.